app/models/calculation_result.rb

```diff
@@ -6,21 +6,10 @@ def initialize(api_response)
     @api_response = api_response
   end
 
-  def eligible?
-    decision == "eligible"
-  end
-
   def decision
     api_response.dig(:result_summary, :overall_result, :result) || "ineligible"
   end
```

Comment:
Must confess this method has always made me slightly nervous - and now more so that we're rending a template based on the API result (which has more valid values than we are rendering, we just know that they will never happen for our inputs). Ideally we should have 2 (or 3) classes so that you can't ask the contribution if you're ineligible (or eligible now?) - but I'm not quite sure how much we should do. A comment and/or limiting the return values might be enough. WDYT?

Reply:
Good point! I think adding a comment and limiting the return values as you suggest is enough for now. I'll do that.